x-pack/plugins/security_solution/public/common/hooks/eql/use_eql_preview.test.ts

```diff
@@ -136,6 +136,30 @@ describe('useEqlPreview', () => {
     });
   });
 
+  // TODO: Determine why eql search strategy returns null for meta.params.body
+  // in complete responses, but not in partial responses
+  it('should update inspect information on partial response', async () => {
+    const mockResponse = getMockResponse();
+    await act(async () => {
+      (useKibana().services.data.search.search as jest.Mock).mockReturnValue(
+        of({
+          isRunning: true,
+          isPartial: true,
+          rawResponse: mockResponse.rawResponse,
+        } as EqlSearchStrategyResponse<EqlSearchResponse<Source>>)
```

Comment:
nit: Instead of 'as casting' which we know is dangerous as it can allow untyped properties to be added, it looks like you can use the of template like so:

```ts
        of<EqlSearchStrategyResponse<EqlSearchResponse<Source>>>({
          isRunning: true,
          isPartial: true,
          rawResponse: mockResponse.rawResponse,
        })
```

Reply:
Updated, thanks!